src/structs.hpp

```diff
@@ -17,6 +17,16 @@ namespace wr
 		std::uint32_t m_id = 0;
 	};
 
+	struct MaterialUVScales
+	{
+		float m_albedo_scale = 1.f
```

Comment:
Consider using 1.0f instead of 1.f. The former seems to be used more often in the codebase.